diff --git a/compiler/catala_web_interpreter.ml b/compiler/catala_web_interpreter.ml
deleted file mode 100644
index cc842a5d..00000000
--- a/compiler/catala_web_interpreter.ml
+++ /dev/null
@@ -1,28 +0,0 @@
-open Driver
-open Js_of_ocaml
-
-let _ =
-  Js.export_all
-    (object%js
-       method interpret
-           (contents : Js.js_string Js.t)
-           (scope : Js.js_string Js.t)
-           (language : Js.js_string Js.t)
-           (trace : bool) =
-         driver
-           (Contents (Js.to_string contents))
-           {
-             Utils.Cli.debug = false;
-             unstyled = false;
-             wrap_weaved_output = false;
-             avoid_exceptions = false;
-             backend = "Interpret";
-             language = Some (Js.to_string language);
-             max_prec_digits = None;
-             trace = false;
-             disable_counterexamples = false;
-             optimize = false;
-             ex_scope = Some (Js.to_string scope);
-             output_file = None;
-           }
-    end)
diff --git a/compiler/dune b/compiler/dune
index 2c5a1996..f6c38809 100644
--- a/compiler/dune
+++ b/compiler/dune
@@ -19,16 +19,6 @@
  (libraries calendar zarith zarith_stubs_js)
  (modules runtime))
 
-(executable
- (name catala_web_interpreter)
- (modes byte js)
- (package catala)
- (public_name catala_web_interpreter)
- (modules catala_web_interpreter)
- (preprocess
-  (pps js_of_ocaml-ppx))
- (libraries catala.driver js_of_ocaml))
-
 (executable
  (name catala)
  (modes native js)
