diff --git a/compiler/catala_web.ml b/compiler/catala_web.ml
deleted file mode 100644
index 31d5289..0000000
--- a/compiler/catala_web.ml
+++ /dev/null
@@ -1,16 +0,0 @@
-open Driver
-open Js_of_ocaml
-
-let _ =
-  Js.export_all
-    (object%js
-       method interpret (contents : Js.js_string Js.t) (scope : Js.js_string Js.t)
-           (language : Js.js_string Js.t) (trace : bool) =
-         driver
-           (Contents (Js.to_string contents))
-           false false false "Interpret"
-           (Some (Js.to_string language))
-           None trace false
-           (Some (Js.to_string scope))
-           None
-    end)
diff --git a/compiler/dune b/compiler/dune
index d192291..994c528 100644
--- a/compiler/dune
+++ b/compiler/dune
@@ -10,16 +10,6 @@
  (libraries calendar zarith zarith_stubs_js)
  (modules runtime))
 
-(executable
- (name catala_web)
- (modes byte js)
- (package catala)
- (public_name catala_web)
- (modules catala_web)
- (preprocess
-  (pps js_of_ocaml-ppx))
- (libraries catala.driver js_of_ocaml))
-
 (executable
  (name catala)
  (modes native)